src/main/java/mate/academy/bookshop/dto/user/UserLoginRequestDto.java

```diff
@@ -1,4 +1,14 @@
 package mate.academy.bookshop.dto.user;
 
-public class UserLoginRequestDto {
+import jakarta.validation.constraints.NotBlank;
+import org.hibernate.validator.constraints.Length;
+
+public record UserLoginRequestDto(
+        @NotBlank
+        @Length(min = 6, max = 24)
```

Comment:
Let's better use jakarta.validation annotations instead of Hibernate specific ones.